src/mokka/src/integration-test/java/pl/hycom/mokka/emulator/mock/MockConfigurationControllerIT.java

```diff
@@ -0,0 +1,180 @@
+package pl.hycom.mokka.emulator.mock;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.web.context.WebApplicationContext;
+import pl.hycom.mokka.Application;
+
+import java.util.UUID;
+import java.util.stream.StreamSupport;
+
+import static io.restassured.RestAssured.expect;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+/**
+ * @author adam.misterski@hycom.pl
+ */
+@DirtiesContext
+@ActiveProfiles("test")
+@AutoConfigureMockMvc
+@AutoConfigureTestDatabase
+@SpringBootTest(classes = Application.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class MockConfigurationControllerIT {
+
+    @Autowired
+    protected WebApplicationContext wac;
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
+    private MockConfigurationRepository mockConfigurationRepository;
```

Comment:
Integration tests are so called "white box" kind of tests. This means they should abstract away from the implementation details. It is like view from client perspective.

So this dependency should be removed - I will review test methods with proper comments.